Prevent instructions that are unsupported by older CPUs from being used

Run backend.d through `sdfmt`, to prevent `make check` from failing

--- a/src/d/llvm/backend.d
+++ b/src/d/llvm/backend.d
@@ -46,14 +46,8 @@ public:
 		}
 
 		targetMachine = LLVMCreateTargetMachine(
-			LLVMGetFirstTarget(),
-			LLVMGetDefaultTargetTriple(),
-			LLVMGetHostCPUName(),
-			LLVMGetHostCPUFeatures(),
-			LLVMCodeGenOptLevel.Default,
-			Reloc,
-			LLVMCodeModel.Default,
-		);
+			LLVMGetFirstTarget(), LLVMGetDefaultTargetTriple(), "x86-64", "",
+			LLVMCodeGenOptLevel.Default, Reloc, LLVMCodeModel.Default, );
 
 		pass = new CodeGen(sema, main, targetMachine, config.debugBuild);
 	}
